_data/navbars/getting-started.yml

```diff
@@ -68,6 +68,8 @@ section:
         path: /getting-started/windows-calico/requirements
       - title: Install Calico for Windows
         path: /getting-started/windows-calico/standard
+    - title: Install on Rancher Kubernetes Engine
```

Comment:
Now that we have Rancher, I believe the nav should mirror what we have in Kubernetes Linux. Like this:

Calico Enterprise for Windows
...
> Kubernetes (instead of "Standard install" at root level)
 - Standard (instead of "Install Calico Enterprise for Windows"
 - Rancher Kubernetes Engine

Reply:
I moved the Calico for Windows on OpenShift doc under `getting-started/windows-calico` too